### ISSUE TITLE
[Minor bug] Invalid chmod flags for scripts/uninstall.sh

### ISSUE THREAD

**Issue by ig-sinicyn** [quaxalber/bluetooth_2_usb#74]
Hi!

```
igors@hid-bridge:~/bluetooth_2_usb/scripts $ ls -l
total 16
-rw-r--r-- 1 igors igors 3473 Dec  9 19:00 build_python_3.11.sh
-rwxr-xr-x 1 igors igors 3828 Dec  9 19:00 install.sh
-rw-r--r-- 1 igors igors 1549 Dec  9 19:00 uninstall.sh
-rwxr-xr-x 1 igors igors 2699 Dec  9 19:00 update.sh
```
looks like x flags are missing for some bash scripts:)

Sorry, I'm kinda busy and cannot do a proper PR for the issue.

**Comment by quaxalber** [comment]
Good find. Will be fixed with next release

**Comment by quaxalber** [comment]
Fixed in [v0.6.5](https://github.com/quaxalber/bluetooth_2_usb/releases/tag/v0.6.5)

Thx!
